lib/base-chips/field_type.hashed-text.js

```diff
@@ -0,0 +1,44 @@
+var Sealious = require('../main.js');
+var Promise = require("bluebird");
+var crypto = require("crypto");
+
+var field_type_hashed_text = new Sealious.ChipTypes.FieldType({
+	name: "hashed-text",
+	extends: "text",
+	get_description: function(context, params){
+		return "Hash text with chosen algorithm (default md5)";
+	},
+	is_proper_value: function(accept, reject, context, params, new_value){
+		var pattern_array = [];
+		if (params instanceof Object){
```

Comment:
Because we don't know if `params` is indeed an object (`{}`) and not string or a number.

But, as we spoke today, I will check the function that validates the `params` value. :)